common/app/implicits/Requests.scala

```diff
@@ -108,8 +108,11 @@ trait Requests {
 
     // dotcom-rendering (DCR) parameters
     lazy val forceDCROff: Boolean = r.getQueryString("dcr").contains("false")
-    lazy val forceDCR: Boolean =
+    lazy val forceDCR: Boolean = {
       r.getQueryString("dcr").isDefined && !forceDCROff // don't check for .contains(true) so people can be lazy
+    }
+    lazy val forceLiveOff: Boolean = r.getQueryString("live").contains("false")
+    lazy val forceLive: Boolean = r.getQueryString("live").isDefined && !forceLiveOff
```

Comment:
Is this easier to fathom?

```suggestion
    lazy val forceLive: Boolean = r.getQueryString("live").contains("true")
```

Reply:
I think the idea is that this allows you to just type `?dcr` in the URL if you want to be lazy

Reply:
Yes exactly, thought I'd do the same so you can do just ?live

Reply:
Not fussed, just seems odd to have 3 options when 2 will do. Being more explicit generally adds clarity and ease of code reading.